layout/js/gages.js

```diff
@@ -107,4 +107,11 @@ var hovertext;
       mapSVG.hideTooltip();
     }
   }
+  
+//  vizlab.showline = function(year) {
+//     var lineId = '#l' + year;
+//     document.getElementById(lineId).setAttribute(class, 'selected-year'//);
```

Comment:
@ldecicco-USGS `selected-year` is the same class used in the other svg, and I think it gets wiped every timestep that the animation plays. Is this the issue? If a different class was created an used, would this still need to be removed?